svdtools/patch.py

```diff
@@ -912,7 +912,7 @@ def collect_in_array(self, rspec, rmod):
             name = rmod["name"]
         else:
             name = rspec[:li] + "%s" + rspec[len(rspec) - ri :]
-        if dimIndex[0] == "0":
+        if not rmod.get("_keep_zero_character", "") and dimIndex[0] == "0":
```

Comment:
```suggestion
        if not rmod.get("_keep_zero_character") and dimIndex[0] == "0":
```

The default is `None` which should evaluate false-y here like `""`, right?

Reply:
Right you are, it is indeed None which is truthy!
Want me to make the same change to the code I used as an example?
https://github.com/stm32-rs/svdtools/blob/7c4bf815b8ba185cb1e92e8e1a4a0c3e1ee46192/svdtools/patch.py#L886

Reply:
I've assumed that you'd want the line 886/1236 changes as well in commit `0dda853 `, let me know if you want me to amend to just fix the new code.